pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -411,6 +411,7 @@ func TestContainerRuntimeConfigCreate(t *testing.T) {
 			f.mccrLister = append(f.mccrLister, ctrcfg1)
 			f.objects = append(f.objects, ctrcfg1)
 
+			f.expectGetMachineConfigAction(mcs1)
```

Comment:
nit: I didn't really dig in to see why this test requires this to be run twice, but it would be nice to denote why in a comment.

Reply:
it's the standard unit pattern in the unit tests that mimick a controller - it's like a framework and we call `expect*` function everytime we want to check that a controller is doing what is supposed to do. I think yes, we could refactor this someday and better document it but adding a comment here doesn't buy us anything now :(

Reply:
I see. Viewing it _is_ confusing from the outside, but I accept your response :slightly_smiling_face:.